curriculum/challenges/english/25-front-end-development/review-react-basics/67487e141bb6a7140a352e12.md

```diff
@@ -0,0 +1,253 @@
+---
+id: 67487e141bb6a7140a352e12
```

Comment:
We also need to add the section for imports and exports because that script was added a few days ago

Reply:
added